tests/testthat/test_wwinference.R

```diff
@@ -60,10 +60,15 @@ test_that("Function to get mcmc options produces the expected outputs", {
   mcmc_options <- get_mcmc_options()
   expected_names <- c(
     "iter_warmup", "iter_sampling",
-    "n_chains", "seed", "adapt_delta", "max_treedepth",
-    "compute_likelihood"
+    "chains", "seed", "adapt_delta", "max_treedepth",
+    "show_messages", "refresh", "save_latent_dynamics", "output_dir",
+    "output_basename", "sig_figs", "parallel_chains", "chain_ids",
+    "threads_per_chain", "opencl_ids", "save_warmup", "thin",
+    "adapt_engaged", "step_size", "metric", "metric_file", "inv_metric",
+    "init_buffer", "term_buffer", "window", "fixed_param", "show_exceptions",
+    "diagnostics", "save_metric", "save_cmdstan_config"
   )
-  # Checkmade doesn't work here for a list, says it must be a character vector
+  # Checkmare doesn't work here for a list, says it must be a character vector
```

Comment:
Typo 
```suggestion
  # Checkmate doesn't work here for a list, says it must be a character vector
```

Also, this surprises me. The following code works for me:

```R
a_list <- list(
   a = 5,
   b = list(x=1, y=2),
   c = data.frame(x=1:5)
)
checkmate::check_names(names(a_list), must.include = c("a", "b", "c"))
```

Reply:
Me too, that might be an old comment when I was using it incorrectly let me see